ffi/input_pocketbook.lua

```diff
@@ -114,15 +254,17 @@ local function translateEvent(t, par1, par2)
             -- i.e., CCW
             genEmuEvent(C.EV_MSC, C.MSC_GYRO, C.DEVICE_ROTATED_COUNTER_CLOCKWISE)
         end
-    else
-        genEmuEvent(t, par1, par2)
     end
     return 0
 end
 
 local raw_keymap -- [EV_KEY] = "Action" supplied by frontend
 local poll_fds, poll_fds_count = nil, 0
 function input:open()
+    -- Don't do this at home, kids, these belong to front.
+    DEBUG = require("dbg")
+    logger = require("logger")
+
```

Comment:
Are these left on purpose?

Reply:
Yeah, I want to guard the logging calls behind verbose debug like Input, so the hackery is necessary so as not to upset the testsuite (I'm assuming this is even covered by the testsuite); but, in any case, not an example to follow; but I didn't want to complicate frontend code just to get a sane logger in here ;).